frontend/src/api.js

```diff
@@ -1,19 +1,37 @@
 import axios from "axios";
-import { LANGUAGE_VERSIONS } from "./contants";
 
 const API = axios.create({
```

Comment:
if we use this a lot (we might), we can refactor this to another folder/file (lib)